benchmarks/auth/python/Dockerfile.python

```diff
@@ -0,0 +1,17 @@
+## First stage: install all dependencies for this workload
```

Comment:
Then we should also change the python dockerfile naming.
We only need `Dockerfile.python`